src/app/pages/StakingPage/Features/DelegationList/__tests__/__snapshots__/ActiveDelegationList.test.tsx.snap

```diff
@@ -592,7 +592,7 @@ exports[`<ActiveDelegationList  /> should match snapshot 1`] = `
                     class="c0"
                     style="display: inline-flex; white-space: nowrap; font-family: "Roboto mono", monospace; letter-spacing: 0;"
                   >
-                    0.0000001
+                    0.00
```

Comment:
If it's not longer an accurate value maybe it's worth adding it to expanded details as it is different value than "Total escrow"?

Reply:
ah it is missing in debonding page; delegation had it under the input field. Added as text now too:
![image](https://github.com/user-attachments/assets/1977ce6d-6972-4e41-94c3-0ed52af466e7)